src/index.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import localForage from 'localforage';
 import { hydrate, render } from 'react-dom';
-import * as Sentry from '@sentry/react';
-import { Integrations } from '@sentry/tracing';
+import * as Sentry from '@sentry/browser';
```

Comment:
Do we need to use `@sentry/browser` and add dependency? I mean can we use the same '@sentry/react' and '@sentry/tracing'